app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ActiveRecord::Base
+  belongs_to :user, class_name: 'user', foreign_key: 'user_id'
```

Comment:
- [ ] Kindly I recommend you to adjust the associations between like and user model Just like the below sample 

```ruby
belongs_to :author, class_name: 'User'
belongs_to :post
```